bin/validate-data-package

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# Script: validate_frictionless_package.sh
+# Description: Bash script to validate a Frictionless Data Package using the Frictionless CLI.
+# Usage: validate_frictionless_package.sh [-v tides_version | -l local_schema_location] [-d dataset_location]
```

Comment:
it looks like there's no `.sh` extension on this script, despite what the usage says.